tests/constraints.txt

```diff
@@ -24,7 +24,8 @@ pytest-html==4.1.1
 pytest-timeout==2.2.0
 jax<=0.4.14
 jaxlib<=0.4.14
-torch>=1.13,<2.3
 kornia==0.7.0
 networkx<=3.1.0
 keras>=2.0.0,<3.0.0
+--extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
If you adding this in constraints.txt it means we can remove it from model tests https://github.com/openvinotoolkit/openvino/blob/fc6af4f4afa16ca8d86b2f99779be12151c43bc4/tests/model_hub_tests/pytorch/requirements.txt#L2

Reply:
done